Packs/CommonScripts/Scripts/CheckPDFEncryptionAndValidity/CheckPDFEncryptionAndValidity.yml

```diff
@@ -0,0 +1,34 @@
+args:
+- default: false
+  description: 'The entry id of the PDF file'
+  isArray: false
+  name: EntryID
+  required: true
+comment: 'Returns if a PDF is valid and if it is encrypted.'
+commonfields:
+  id: CheckPDFEncryptionAndValidity
+  version: -1
+enabled: false
+name: CheckPDFEncryptionAndValidity
+outputs:
+- contextPath: File.IsEncrypted
+  description: 'True if PDF file is encrypted, False else'
+  type: Boolean
+- contextPath: File.IsValid
+  description: 'True if PDF file is readable, False else'
+  type: Boolean
+- contextPath: File.EntryID
+  description: 'The entry id of the file'
```

Comment:
```suggestion
  description: 'True if PDF file is encrypted, False otherwise.'
  type: Boolean
- contextPath: File.IsValid
  description: 'True if PDF file is readable, False otherwise.'
  type: Boolean
- contextPath: File.EntryID
  description: 'The entry ID of the file.'
```